support/grunt/config.ts

```diff
@@ -65,19 +65,13 @@ export const copy = {
 
 export const intern = {
 	options: {
-		runType: 'client',
 		reporters: [
-			'Console', 'LcovHtml'
+			'runner', 'htmlcoverage'
 		]
 	},
 	unit: {
 		options: {
-			config: '_build/tests/intern'
-		}
-	},
-	integration: {
-		options: {
-			config: '_build/tests/intern-integration'
+			config: 'intern.json'
```

Comment:
How do we run integration tests? `index.ts` still has a line for them:

```
grunt.registerTask('integration', [ 'dev', 'intern:integration' ]);
```

Reply:
I've addressed this in a local commit